README.md

```diff
@@ -1 +1,57 @@
-# se452-group7-project
\ No newline at end of file
+# se452-group7-project
+
+This repository is the group project for SE453 - Group 7 - Fall 2024   
+
+**Project Scope Area**
+---
+E-commerce Application
+We are building a simple shopping application where users can create accounts, browse different product categories, place orders, and make payments within the app.
+
+**Project Members**
+---
+| Name              | Functional Area |
+|---------------    | --------------- |
+| Bhumika Ramesh    | Account Management |
+| Rachana           | |
+| Saalim            | |
+| Likith            | |
+
+**Initial key features for each of the members of the team**
+---
+Bhumika Ramesh
+
+For the account management feature, I will focus on implementing user registration and authentication, allowing users to create accounts securely. This will include functionality for password recovery and reset, ensuring that users can easily regain access if needed. Additionally, I will develop profile management capabilities allowing users to update their personal information.
+
+Rachana
+
+Likith
+
+Saalim
+
+**Github location**
+---
+
+
+**Conflict Resolution**
+---
+
+
+**Communication Mechanism**
+---
+Primary - WhatsApp for day-to-day communication.
+Secondary - Email for official communication and GitHub for issue tracking.
+Meetings - Weekly virtual meetings via Teams to review progress and assign new tasks.
+
+**Meeting journal**
+---
+
+**Decision Made**
+---
+1. IDE - VS Code
+2. Dependency Management - Lombok
+3. Code - Java
+4. Configuration Management - 
```

Comment:
We can use yaml for configuration management

Reply:
Sure.